log4net.build

```diff
@@ -1748,48 +1748,64 @@ limitations under the License.
       </csc>
     </if>
   </target>
-  <!-- these targets build lognet assemblies against .NET Core (.net standard 1.0.4)-->
-  <target name="compile-netstandard" description="Builds .NET Core">
+  <!-- these targets build lognet assemblies against .NET Standard 1.3 and 2.0 -->
+  <target name="compile-netstandard" description="Builds .NET Standard">
```

Comment:
I would prefer this to become the target `compile-netstandard-1.3` and a second target should be added named `compile-netstandard-2.0`. Otherwise too many different targets and builds are done by one target and maintaining it becomes a hard task.